modules/editor/message/MessageEditor.tsx

```diff
@@ -40,35 +83,34 @@ export function MessageEditor(props: MessageEditorProps) {
 
   return useObserver(() => (
     <Stack gap={16}>
-      <div>
-        <PrimaryContentEditor message={message} form={form} />
-        <ErrorWrapper>
-          <InputError
-            error={
-              message.embedLength > 6000
-                ? "Embeds exceed 6000 character limit"
-                : undefined
-            }
-          />
-        </ErrorWrapper>
-      </div>
-      {message.embeds.map((embed, index) => (
-        <EmbedEditor
-          key={embed.id}
-          embed={embed}
-          form={form.repeatingForm("embeds").index(index)}
-        />
-      ))}
-      <div>
+      <PrimaryContentEditor message={message} form={form} />
+      <EmbedGroup message={message} form={form} />
+      <InputField
+        id={`_${message.id}_url`}
```

Comment:
Changing id to match the rest of the project
```suggestion
        id={`_${message.id}_reference`}
```